src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> ticketsId;
+    private LocalDateTime timeOrder;
```

Comment:
Consider renaming 'timeOrder' to 'orderTime' for better readability and understanding of its purpose.